src/diric.jl

```diff
@@ -0,0 +1,55 @@
+export diric
+
+"""
+    kernel = diric(Ω::Real, n::Int; atol)
+
+Dirichlet kernel, also known as periodic sinc function,
+where `n` should be a positive integer.
+Returns `sin(Ω * n/2) / (n * sin(Ω / 2))` typically,
+but `±1` when `Ω` is a multiple of 2π.
+
+In the usual case where 'n' is odd, the output is equivalent to
+``1/n \\sum_{k=-(n+1)/2}^{(n+1)/2} e^{i k Ω}``,
```

Comment:
```suggestion
``1/n \\sum_{k=-(n-1)/2}^{(n-1)/2} e^{i k Ω}``,
```
Alternatively, `e^{i (n-1) Ω/2}/n \\sum_{k=0}^{n-1} e^{i k Ω}` could be used, which would also hold for even `n`.

Reply:
Doh!  Good catch.

Reply:
BTW I started making other edits before noticing this suggestion.  I did incorporate the suggestion but I'm not enough of a git wizard to know how to "commit" your version.  Sorry about that.

Reply:
No worries.